monero-outreach-docs/en/storyboard/wallet_quickstart_en.md

```diff
@@ -0,0 +1,57 @@
+
+Light wallets come in two types: view-key-protecting and view-key-sharing. View-key-protecting light wallets keep your view key completely private, and as a consequence have to download the blockchain, or at least the portion of it later than the wallet’s creation date, to calculate the Monero balance in the wallet. View-key-sharing light wallets, on the other hand, share the view key with the server running the remote node to calculate the wallet’s incoming balance on the server. These efficient wallets are sometimes called wallet apps. Their sharing of the view key reduces privacy if the server running the view-key-aware remote node is not trusted because it allows observation of incoming transactions. Note, however, that sometimes wallet holders share the view key for auditing anyway—it does not allow transferring Monero.
+
+For maximum security—security even if your primary device is compromised by malware—specialized hardware wallets can be used. Hardware wallets are small interactive electronic devices that store your private keys and communicate with software on the primary device using only public information. Hardware wallets give extra security for a typical price of $50-150.
+
+So, with all this in mind, consider the following options: If you want to run your own full node, a good choice is the official Monero GUI or CLI wallet. The former provides a graphical user interface, and the latter a textual interface. Both can be configured to either run a full node or connect to a remote node in a privacy enhanced way. These can be downloaded from [getmonero.org/downloads](https://getmonero.org/downloads/). For Android devices, Monerujo, at [monerujo.io](https://www.monerujo.io/), or for iOS, Cake Wallet, at [cakewallet.io](https://cakewallet.io/), are recommended view-key-protecting light wallets. If you would like an view-key-sharing light wallet, browser-based MyMonero, at [mymonero.com](https://mymonero.com/), is recommended. The Edge Wallet, at [edge.app](https://edge.app/), is another solid view-key-sharing light-wallet option for both Android and iOS.
+
+When choosing among these, keep in mind that most have an extra feature in that—the exception being the Edge Wallet—they can be configured to connect to either a third-party remote server or a server you run with a node. This is illustrated in the table.
+
+## Official Wallets
+
++ [getmonero.org/downloads](https://www.getmonero.org/downloads)
+If you want to run a full node, a good choice is the official Monero GUI or CLI wallet. The former provides a graphical user interface, and the latter a textual interface.
+
+## Mobile Wallets
+
++ [cakewallet.io](https://cakewallet.io/)
+Cake Wallet, a view-key-protecting light wallet, is available for iOS devices.
```

Comment:
This 2 string will be on the same line, is this the format intended? Same for the following strings with the same style

Reply:
No, they are different, let me add trailing spaces.

Reply:
Thank you for the notice. I always had trouble with those lines and instead of adding a trailing space I added a space between lines. Thank you @erciccione, I'm still learning a lot each day :).